### PR TITLE
Added insert of updated preferences in onboarding

### PR DESCRIPTION
**Pull request by dyampols**
- Now if user updates their preference / interest the associated activity id will be populated in the database 
- Since update fields are not required on the profile page, I added handling if the user does not input any values into the text fields if they do not want to update them. When I was working on this, when I did not fully fill out the form it would not actually update in the database even though it would pass successfully in the front end. Now this is not an issue.
- Similar strategy to this PR - https://github.com/lkdfount/msci342project/pull/77
- Created a change to a past unit test

### CODE REVIEW

File: travvy/src/components/Profile.vue [hunk 1]
```diff
@@ -73,21 +73,45 @@
...
+        if(!this.gender){
+          this.gender = this.$store.state.gender
+        }
+        if(!this.instagram_username){
+          this.instagram_username = this.$store.state.instagram
+        }
+        if(!this.preferred_activity_type){
+          this.preferred_activity_type = this.$store.state.activity
+        }
+
```
Comment (emilynairnn) [inline travvy/src/components/Profile.vue:107]: I'm glad you were able to add the ability for users to have their attractions personalized on the profile page as well! Nice to give users the opportunity to change their preferred activity type.

**Review by emilynairnn — APPROVED** [review]
Looks good to me! Great work.

File: travvy/src/components/Profile.vue (hunk 1 above)
Comment (dyampols) [reply travvy/src/components/Profile.vue:107]: Thanks!

**Review by lucasungaretti — APPROVED** [review]
LGTM

**Pull request merged**